convert2rhel/initialize.py

```diff
@@ -77,4 +77,11 @@ def run():
 
     from convert2rhel import main
 
-    sys.exit(main.main())
+    try:
+        retval = 1
+        with applock.ApplicationLock("convert2rhel") as _lock:
+            retval = main.main()
+    except applock.ApplicationLockedError:
+        sys.stderr.write("Another copy of convert2rhel is running.\n")
```

Comment:
This is a difference in philosophy; main() has a lot of code behind it, and I don't think anyone could swear that we're accounting for all the exceptions that could possibly occur, so my tendency would be to assume that things have failed until they're known to have succeeded, so you can count on $? being set correctly.

Reply:
To finish this out, Joe and Rodolfo talked about this and the suggestion was added.  I wasn't present for that but my guess is that the discussion concluded if there's an error in `main()` that raises an unhandled exception, then we will exit with exit code 1 and a traceback because that's Python's behaviour with an unhandled exception.  So `retval` doesn't need to be initialized to 1 in this code to handle those unanticipated cases.